Formula/grails.rb

```diff
@@ -1,15 +1,18 @@
 class Grails < Formula
   desc "Web application framework for the Groovy language"
   homepage "https://grails.org"
-  url "https://github.com/grails/grails-core/releases/download/v4.0.2/grails-4.0.2.zip"
-  sha256 "897496c61b6d87fb61e5adf3d0ba27783cb779c4492930d1c8ad7e603bf9c267"
+  url "https://github.com/grails/grails-core/releases/download/v4.0.3/grails-4.0.3.zip"
+  sha256 "4a1b219451a4a64d0236ad1e1b6e1946ca916258db069c526eb751b1f9dbed80"
 
   bottle :unneeded
 
+  depends_on "openjdk"
+
   def install
     rm_f Dir["bin/*.bat", "bin/cygrails", "*.bat"]
     libexec.install Dir["*"]
-    bin.install_symlink Dir["#{libexec}/bin/*"]
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files libexec/"bin", :JAVA_HOME => Formula["openjdk"].opt_prefix
```

Comment:
This would allow the user to override `JAVA_HOME`:

```suggestion
    bin.env_script_all_files libexec/"bin", :JAVA_HOME => "${JAVA_HOME:-#{Formula["openjdk"].opt_prefix}}"
```